emails/index.pug

```diff
@@ -13,8 +13,8 @@ html
         main
             .info 
                 div
-                p This is an automated email. Do not reply to this. You received this email <br/> because you are registered with SpaceToStudy at <br/>
+                p This is an automated email. Do not reply to this. You received this email <br/> because you are registered at SpaceToStudy with email: <br/>
```

Comment:
with a #{email} at SpaceToStudy - or find some workaround, your version doesn't look pretty
also consider removing <br/> if it can help

Reply:
changed